src/components/section/Section.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from 'react';
+import Box from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+import { BoxProps } from '../box/Box';
+
+type Color = 'white' | 'neutral' | 'mint' | 'violet' | 'ruby' | 'gold' | 'aqua';
+type Size = 'small' | 'medium' | 'large';
+
+const SIZES: { [K in Size]: number } = {
```

Comment:
```suggestion
const SIZES: Record<Size, number> = {
```

This is a bit easier to understand IMO